src/Microsoft.DotNet.Wpf/src/WpfGfx/tools/csp/Project.cs

```diff
@@ -96,7 +96,7 @@ Parameters parameters
 
             // Add some implicit referenced assemblies. 
 
-            referencedAssemblies.Add(Path.Combine(parameters.ClrDir, "System.dll"));
+            referencedAssemblies.Add(string.IsNullOrEmpty(parameters.ClrDir) ? "System.dll" : Path.Combine(parameters.ClrDir, "System.dll"));
```

Comment:
It says in the usage of this project that `parameters.ClrDir` is optional but it failed here.